src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -76,6 +77,37 @@ g.test('visibility')
     }, !success);
   });
 
+g.test('visibility,VERTEX_shader_stage_storage_texture_access')
+  .desc(
+    `
+  Test that a validation error is generated if the access value is 'write-only' when the
+  visibility of the entry includes VERTEX.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('shaderStage', kShaderStageCombinations)
+      .beginSubcases()
+      .combine('access', [undefined, ...kStorageTextureAccessValues])
+  )
+  .fn(async t => {
+    const { shaderStage, access } = t.params;
+
+    const success = !(access === 'write-only' && shaderStage & GPUShaderStage.VERTEX);
```

Comment:
According to the spec, if entry.visibility includes VERTEX, entry.storageTexture?access must not be "write-only". So, that's why I set `success` with this condition. But, when I ran this test, the test generated a validation error with the below error message. It's likely storageTexture only could be used with ShaderStage::(Fragment|Compute), not ShaderStage::Vertex.

`  - EXCEPTION: Unexpected validation error occurred: storageTexture bindings cannot be used with a visibility of ShaderStage::Vertex. Only ShaderStage::(Fragment|Compute) are allowed.`

However, I couldn't find this behavior in the specification. I wonder if we need to update Chromium/dawn to meet this requirement. If I miss finding the current behavior(storageTexture bindings cannot be used with a visibility of ShaderStage::Vertex) in the specification, please correct me.

Reply:
`"write-only"` is actually the *only* possible value of `access`. It's the default, so the case of `undefined` is actually also `"write-only"`. (The reason we still like to test both even when there's a default, is just so we can test the defaulting behavior.)
https://gpuweb.github.io/gpuweb/#dictdef-gpustoragetexturebindinglayout

So I think the implementation is correct.

Reply:
If you implement the same defaulting here, like so:
```suggestion
    const success = !((access ?? 'write-only') === 'write-only' && shaderStage & GPUShaderStage.VERTEX);
```
then this test might pass.

Reply:
Thank you for explaining it. And, I update the test with the suggestion.